pkg/csr/csr.go

```diff
@@ -271,9 +271,6 @@ func hasUsages(csr *certificates.CertificateSigningRequest, usages []certificate
 	if csr == nil {
 		return false
 	}
-	if len(usages) != len(csr.Spec.Usages) {
```

Comment:
Shouldn't we still be checking that csr.Spec.Usages contains the required 2 usages (certificates.UsageDigitalSignature, certificates.UsageServerAuth)?

Reply:
Thanks, my reasoning here is to avoid CSR related issues from popping up if we are too lax. Seems upstream is doing exact checking as well